libs/dh/metering-point/feature-overview/src/components/measurement/dh-measurements-all-years.component.ts

```diff
@@ -0,0 +1,131 @@
+
+import { WattSupportedLocales } from '@energinet-datahub/watt/date';
+import { WattDataTableComponent } from '@energinet-datahub/watt/data';
+import { VaterUtilityDirective } from '@energinet-datahub/watt/vater';
+import { WATT_TABLE, WattTableColumnDef, WattTableDataSource } from '@energinet-datahub/watt/table';
+
+import {
+  Quality,
+  GetAggregatedMeasurementsForAllYearsDocument,
+} from '@energinet-datahub/dh/shared/domain/graphql';
+import { query } from '@energinet-datahub/dh/shared/util-apollo';
+import { getPath, MeasurementsSubPaths } from '@energinet-datahub/dh/core/routing';
+
+import { AggregatedMeasurementsForAllYears } from '../../types';
+import { dhFormatMeasurementNumber } from '../../utils/dh-format-measurement-number';
+
+@Component({
+  selector: 'dh-measurements-all-years',
+  imports: [TranslocoDirective, WATT_TABLE, WattDataTableComponent, VaterUtilityDirective],
+  template: `
+    <watt-data-table
+      vater
+      inset="ml"
+      [enableSearch]="false"
+      [enableCount]="false"
+      [error]="query.error()"
+      [ready]="query.called()"
+      [enablePaginator]="false"
+      *transloco="let t; read: 'meteringPoint.measurements'"
+    >
+      <watt-table
+        *transloco="let resolveHeader; read: 'meteringPoint.measurements.columns'"
+        [resolveHeader]="resolveHeader"
+        [columns]="columns"
+        [stickyFooter]="true"
+        [dataSource]="dataSource"
+        [loading]="query.loading()"
+        sortDirection="desc"
+        [sortClear]="false"
+        (rowClick)="navigateToYear($event.year)"
+      >
+        <ng-container *wattTableCell="columns.month; let element">
+          {{ element.year }}
+        </ng-container>
+
+        <ng-container *wattTableCell="columns.currentQuantity; let element">
+          @if (element.quality === Quality.Estimated) {
+            ≈
+          }
+          {{ formatNumber(element.quantity) }}
+        </ng-container>
+      </watt-table>
+    </watt-data-table>
+  `,
+})
+export class DhMeasurementsAllYearsComponent {
+  private router = inject(Router);
+  private route = inject(ActivatedRoute);
+  private transloco = inject(TranslocoService);
+  private locale = inject<WattSupportedLocales>(LOCALE_ID);
+  private sum = computed(() =>
+    this.formatNumber(this.measurements().reduce((acc, x) => acc + x.quantity, 0))
+  );
+  private measurements = computed(() => this.query.data()?.aggregatedMeasurementsForAllYears ?? []);
+  meteringPointId = input.required<string>();
+  query = query(GetAggregatedMeasurementsForAllYearsDocument, () => ({
+    variables: { meteringPointId: this.meteringPointId() },
+  }));
+
+  Quality = Quality;
+
+  columns: WattTableColumnDef<AggregatedMeasurementsForAllYears> = {
+    month: {
+      accessor: 'year',
+      size: 'min-content',
+      footer: { value: signal(this.transloco.translate('meteringPoint.measurements.sum')) },
+    },
+    currentQuantity: {
+      accessor: 'quantity',
+      align: 'right',
```

Comment:
question: wouldn't this push the text all the way to the right? Maybe there needs to be a "spacer" column.

Reply:
You are right, fixed now